static/app/views/monitors/components/monitorForm.tsx

```diff
@@ -494,9 +495,7 @@ function MonitorForm({
               {monitor?.config.alert_rule_id && (
                 <AlertLink
                   priority="muted"
-                  to={normalizeUrl(
-                    `/alerts/rules/${monitor.project.slug}/${monitor.config.alert_rule_id}/`
-                  )}
+                  to={`/organizations/${organization.slug}/alerts/rules/${monitor.project.slug}/${monitor.config.alert_rule_id}/`}
```

Comment:
wait does it not need normalizeUrl anymore?

Reply:
links are normalized for you